### ISSUE TITLE
Ability to customise the automatic emails to staff. 

### ISSUE THREAD

**Issue by victoriawhitmore** [snipe/snipe-it#7143]
**Server (please complete the following information):** - Hosted
 - Snipe-IT Version  v4.7.4 - build 4107 (master)

**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
I would like to be able to customise the emails that go out to staff when checking in or out an asset.  That way we could give more instruction on how to access the site and what to use for their login details.  I would also like to add a link to our IT policy in the email when they have to sign for an asset as at the moment it says to agree to the policy but I get a lot of users asking where it is or how to get to the policy.  It would be great to have that in the email for them. 

Thank you for taking the time to listen to my request. 

Kind Regards, 

Victoria.

**Comment by victoriawhitmore** [comment]
Yes please keep this open as I would still like to see this added in the future updates.

**Comment by victoriawhitmore** [comment]
Yes please keep this open as I would still like to see it added to an update.

**Comment by victoriawhitmore** [comment]
Yes please keep this open as I would still like to see this added.

**Comment by victoriawhitmore** [comment]
Yes, this is still relevant.  I would like to be able to customise the message sent to staff when receiving an item as we would like to add a link to our IT policy.

**Comment by victoriawhitmore** [comment]
Yes this is still relevant

**Comment by victoriawhitmore** [comment]
Still Relevant

**Comment by victoriawhitmore** [comment]
Still Relevant

**Comment by victoriawhitmore** [comment]
Yes still relevant

**Comment by snipe** [comment]
This is not something on the short-term roadmap. You can always work around this by adding additional information to the EULA field which goes out in emails.

**Comment by snipe** [comment]
Not seeing this on the roadmap TBH. The amount of footguns that would happen if we did this would be huge, and our ability to support/debug users would be too diminished.
